fvm/env.go

```diff
@@ -552,13 +529,22 @@ func (e *hostEnv) ImplementationDebugLog(message string) error {
 	return nil
 }
 
-// Transaction Environment
+func (e *hostEnv) HasContractUpdates() bool {
```

Comment:
@m4ksio this is the method to be used, should only use it before env.Commit() call